src/alley/wp/types/interface-serialized-blocks.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Serialized_Blocks interface file
+ *
+ * @package wp-type-extensions
+ */
+
+namespace Alley\WP\Types;
+
+/**
+ * Describes an object with blocks that can be serialized.
+ */
+interface Serialized_Blocks {
+	/**
+	 * Serialized blocks.
+	 *
+	 * @return string
+	 */
+	public function serialized_blocks(): string;
```

Comment:
```suggestion
	public function serialize_blocks(): string;
```
should this be `serialize_blocks()`?

Reply:
Negative. The naming is meant to reinforce what implementing classes are responsible for providing, not how. In this instance, a class is responsible for providing a serialized block string, but it might offload the actual serialization to another class, use an already-serialized string that was provided in the constructor, etc.